### ISSUE TITLE
IPv6 Pings

### ISSUE THREAD

**Issue by NogginTheNog** [vectronic/homebridge-ping-hosts#16]
Any chance of adding an option for IPv6 pings?

**Comment by vectronic** [comment]
Yes. I’ve just looked at the underlying library in use and it is supported. I will work on this when I get a chance in the next week (or two).

**Comment by vectronic** [comment]
Version 3.0.1 should work, use a `ipv4_address` or `ipv6_address` instead of the now deprecated `host` property.

**Comment by NogginTheNog** [comment]
Working a treat, thanks.
